assets/templates/partials/gtm-data-layer.tmpl

```diff
@@ -35,6 +35,14 @@ dataLayer = [{
         "numberOfResults": {{.Count}},
         "resultsPage": {{.Pagination.CurrentPage}},
     {{ end }}
+    {{ if eq .Type "ReleasePage" }}
+        "release-status": {{.ReleasePageData.ReleaseStatus}},
+        "release-date": {{.ReleasePageData.ReleaseDate}},
+        "release-time": {{.ReleasePageData.ReleaseTime}},
+        "release-date-status": {{.ReleasePageData.ReleaseDateStatus}},
+        "contact-name": {{.ReleasePageData.ContactName}},
+        "next-release-date": {{.ReleasePageData.NextReleaseDate}},
```

Comment:
These are quite granular in detail, what would be useful to see the requirements. Also, as these are very service specific it might be worth pushing to the `dataLayer` within your service as the `dp-renderer` is global